target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst

```diff
@@ -1,100 +1,101 @@
-me/ryanhamshire/GPFlags/util/VersionControl.class
```

Comment:
Can you please revert the changes to this class.
This class should not have been pushed to your repo/PR. I'm surprised it did since `target` is part of the gitignore